.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: java
-jdk: oraclejdk8
```

Comment:
until I changed this travis was unhappy as recently oracle license has been killing some builds

Reply:
Thanks, we prefer jdk8 for now